General_Statistics.js

```diff
@@ -0,0 +1,14 @@
+// Sample data (replace with real data from the logging page)
+const data = [
+    { x: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11], y: [3, 5, 2, 7, 4, 6, 1, 8, 9, 10, 11], name: 'Person 1' },
+    // Add more data objects for other people
```

Comment:
Here again the same thing, flip the objects around, so an array of shotObjects is owned by the personObject

Reply:
Or maybe the plotly api needs it in this format?